jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -171,8 +172,23 @@ def process(self, input_data):
 
             state = 'COMPLETE'
             if self.input_container:
+                if not self.save_intermediate_results:
+                    self.log.info("The following files will be deleted since save_intermediate_results=False:")
```

Comment:
You could perhaps change these log statements (here and below in lines 187 and 191) to be debug level, instead of info. It's not really necessary to see these messages during routine processing.

Reply:
makes sense. Done